includes/polling/os/radlan.inc.php

```diff
@@ -1,10 +1,21 @@
 <?php
 
-$version  = snmp_get($device, 'rndBrgVersion.0', '-Ovq', 'RADLAN-MIB');
-$hardware = str_replace('ATI', 'Allied Telesis', $device['sysDescr']);
+// sysDescr.0 = STRING: "ATI AT-8000S"
+// sysDescr.0 = STRING: 48-port 10/100/1000 Ethernet Switch 
+// sysDescr.0 = STRING: 24-port 10/100/1000 Ethernet Switch with PoE
+
+$hardware = snmp_getnext($device, '.1.3.6.1.2.1.47.1.1.1.1.2.64', '-OsvQU', '+ENTITY-MIB');
+$version  = snmp_get($device, '.1.3.6.1.4.1.89.2.4.0', '-OsvQU', '+SNMPv2-SMI');
+$serial   = snmp_get($device, 'ENTITY-MIB::entPhysicalSerialNum.64', '-OsvQU', '+ENTITY-MIB');
```

Comment:
Don't need `ENTITY-MIB::` as you specify it after anyway

Reply:
Have fixed this up for next commit